src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,26 @@
+package lotto.model;
+
+import java.util.List;
+
+public class Lotto {
+
+    private final LottoNumbers numbers;
+
+    public Lotto(List<Integer> numbers) {
+        this.numbers = new LottoNumbers(numbers);
+    }
```

Comment:
```suggestion
    public Lotto(LottoNumbers LottoNumbers) {
        this.numbers = LottoNumbers;
    }
```
생성자의 인자로 `LottoNumbers`이 아닌, `List<Integer>` 타입을 받으신 이유가 있으실까요?

Reply:
앗 수정했습니다🤣🤣